File: .php
<!-- migrate 1 -->
Modify tickets table

use Illuminate\Database\Migrations\Migration;
use Illuminate\Database\Schema\Blueprint;
use Illuminate\Support\Facades\Schema;

class ModifyTicketsTableIdColumn extends Migration
{
    public function up()
    {
        Schema::table('tickets', function (Blueprint $table) {
            // Drop the existing primary key
            $table->dropPrimary();
            
            // Modify the id column
            $table->bigIncrements('id')->change();
        });
    }

    public function down()
    {
        Schema::table('tickets', function (Blueprint $table) {
            // If you need to revert, you might want to set it back to how it was
            // This assumes the original was an unsigned integer
            $table->integer('id')->unsigned()->change();
            
            // Re-add the primary key
            $table->primary('id');
        });
    }
}

<!-- migrate 2 -->

'sto' => 'required|string|in:BJA,CWD,MJY,PDL,PNL,SOR,RJW,CCL,BTJ,CKW,CCL,CPT,CSA,GNH,LEM,RCK',

File: .py
# List kode yang ingin diubah
codes = ['BJA', 'CWD', 'MJY', 'PDL', 'PNL','SOR','RJW','CCL','BTJ','CKW','CCL','CPT','CSA','GNH','LEM','RCK']

# Fungsi untuk mengubah format setiap kode menjadi <option>
def generate_options(codes):
    options = []
    for code in codes:
        option = f'<option value="{code}" {{ old('STO', $ticket->STO) == '{code}' ? 'selected' : '' }}></option>'
                    
        options.append(option)
    return options

# Hasilkan dan cetak opsi
generated_options = generate_options(codes)
for option in generated_options:
    print(option)